src/oc_erchef/apps/oc_chef_authz/src/oc_chef_authz_acl_constraints.erl

```diff
@@ -48,8 +48,9 @@ acl_checks() ->
     fun check_admins_group_removal_from_grant_ace/5
   ].
 
--spec check_admins_group_removal_from_grant_ace(binary(),binary(), atom(),binary(),tuple())
-      -> false | {true, attempted_admin_group_removal_grant_ace}.
+
+-spec check_admins_group_removal_from_grant_ace(binary(), binary(), atom(), binary(), tuple())
+      -> error | false | {true, attempted_admin_group_removal_grant_ace}.
```

Comment:
Hmm it's been a while, so I don't see where the `error` comes from? Is that a default if it's `no_match` or something...?

Reply:
Nice catch, let me check on this one locally.  I think this might be a speculative change that was made when debugging that never got removed after the real error was found.